app.py

```diff
@@ -344,15 +352,27 @@ def generate_graph(street, direction, day_type='Weekday', period='AMPK'):
              className='row'),
     html.Div(id=STATE_DIV_ID, style={'display': 'none'}, children=serialise_state(INITIAL_STATE)),
     html.Div(id=SELECTED_STREET_DIV, style={'display': 'none'}, children=[STREETS[0]])
-    ])
+])
+
+#CSS
+
+app.css.config.serve_locally= True
+app.scripts.config.serve_locally = True
+
+@app.server.route('/css/<path:path>')
```

Comment:
Can you add a docstring to this function with a reference to where you were inspired to get it?